apis/apps/v1alpha1/componentdefinition_types.go

```diff
@@ -101,6 +101,12 @@ type ComponentDefinitionSpec struct {
 	// +kubebuilder:validation:Required
 	Runtime corev1.PodSpec `json:"runtime"`
 
+	// User-defined environment variables.
+	// These can be used as env vars for pods or actions, or to render the templates of config or script.
+	// Cannot be updated.
+	// +optional
+	Env []EnvVar `json:"env,omitempty"`
```

Comment:
Here, the priority relationship between EnvVar and Pod.Container.[Env | EnvFrom] should be clarified.